<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsaWindow</class>
 <widget class="QMainWindow" name="MsaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="horizontalSplitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QSplitter::handle:horizontal
{
  background: white;
  border-left: 1px dotted #000;
}
QSplitter::handle:horizontal:hover
{
background: #ddd;
}</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="EventSignalGraphicsView" name="labelGraphicsView">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::NoAnchor</enum>
        </property>
       </widget>
       <widget class="NativeMsaView" name="msaView" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QAbstractScrollArea {
  border: 0 solid #000;
}</string>
        </property>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="logoTab">
        <attribute name="title">
         <string>Logo</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGraphicsView" name="logoGraphicsView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="renderHints">
            <set>QPainter::Antialiasing</set>
           </property>
           <property name="transformationAnchor">
            <enum>QGraphicsView::NoAnchor</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAlignment">
    <property name="title">
     <string>Alignment</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionTrimRowsLeft"/>
    <addaction name="actionTrimRowsRight"/>
    <addaction name="actionExtendRowsLeft"/>
    <addaction name="actionExtendRowsRight"/>
    <addaction name="actionLevelRowsLeft"/>
    <addaction name="actionLevelRowsRight"/>
    <addaction name="actionCollapseLeft"/>
    <addaction name="actionCollapseRight"/>
    <addaction name="separator"/>
    <addaction name="actionRemoveAllGapColumns"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionDeselectAll"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menu_ShowHide">
     <property name="title">
      <string>Show / Hide</string>
     </property>
    </widget>
    <addaction name="menu_ShowHide"/>
   </widget>
   <widget class="QMenu" name="menu_Visualizations">
    <property name="title">
     <string>&amp;Visualizations</string>
    </property>
    <widget class="QMenu" name="menu_Colors">
     <property name="title">
      <string>&amp;Colors</string>
     </property>
    </widget>
    <addaction name="menu_Colors"/>
   </widget>
   <addaction name="menuAlignment"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Visualizations"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="action_SelectTool"/>
   <addaction name="action_HandTool"/>
   <addaction name="action_ZoomTool"/>
   <addaction name="action_GapTool"/>
  </widget>
  <widget class="QDockWidget" name="subseqTableDockWidget">
   <property name="minimumSize">
    <size>
     <width>113</width>
     <height>118</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Sequence Table</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTableView" name="subseqTableView">
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>22</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/aliases/images/icons/document-save</normaloff>:/aliases/images/icons/document-save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionTrimRowsLeft">
   <property name="text">
    <string>Trim Rows Left</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/aliases/images/icons/edit-undo</normaloff>:/aliases/images/icons/edit-undo</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/aliases/images/icons/edit-redo</normaloff>:/aliases/images/icons/edit-redo</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionTrimRowsRight">
   <property name="text">
    <string>Trim Rows Right</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionExtendRowsLeft">
   <property name="text">
    <string>Extend Rows Left</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionExtendRowsRight">
   <property name="text">
    <string>Extend Rows Right</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionLevelRowsLeft">
   <property name="text">
    <string>Level Rows Left</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionLevelRowsRight">
   <property name="text">
    <string>Level Rows Right</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionRemoveAllGapColumns">
   <property name="text">
    <string>Remove All Gap Columns</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionCollapseLeft">
   <property name="text">
    <string>Collapse Selection Left</string>
   </property>
   <property name="shortcut">
    <string>,</string>
   </property>
  </action>
  <action name="actionCollapseRight">
   <property name="text">
    <string>Collapse Selection Right</string>
   </property>
   <property name="shortcut">
    <string>.</string>
   </property>
  </action>
  <action name="action_SelectTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/aliases/images/icons/edit-select</normaloff>:/aliases/images/icons/edit-select</iconset>
   </property>
   <property name="text">
    <string>Select Tool</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_HandTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/aliases/images/cursors/hand-tool</normaloff>:/aliases/images/cursors/hand-tool</iconset>
   </property>
   <property name="text">
    <string>Hand Tool</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_ZoomTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/aliases/images/icons/zoom-tool</normaloff>:/aliases/images/icons/zoom-tool</iconset>
   </property>
   <property name="text">
    <string>Zoom Tool</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_GapTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/aliases/images/icons/gap-tool</normaloff>:/aliases/images/icons/gap-tool</iconset>
   </property>
   <property name="text">
    <string>Insert Gaps Tool</string>
   </property>
   <property name="toolTip">
    <string>Insert gap columns</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/aliases/images/icons/edit-select-all</normaloff>:/aliases/images/icons/edit-select-all</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Selects the entire alignment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDeselectAll">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/aliases/images/icons/deselect-all</normaloff>:/aliases/images/icons/deselect-all</iconset>
   </property>
   <property name="text">
    <string>Deselect All</string>
   </property>
   <property name="toolTip">
    <string>Clears any selection </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/aliases/images/icons/document-close</normaloff>:/aliases/images/icons/document-close</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NativeMsaView</class>
   <extends>QWidget</extends>
   <header>gui/widgets/NativeMsaView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EventSignalGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>gui/widgets/EventSignalGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
